app/views/content_items/coming_soon.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= content_for :page_class, "coming-soon" %>
+<%= content_for :title, "Coming soon - GOV.UK" %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Coming soon' %>
+
+    <p class="summary">
+      This document will be published on<br />
+      <%= content_tag(:abbr, class: 'publish_time', title: @content_item.iso8601_publish_time) do %>
```

Comment:
The `title` was just so we followed the [microformat datetime pattern](http://microformats.org/wiki/datetime-design-pattern). We shouldn't need to add the title anymore.